stock_picking_backorder_strategy/models/stock_picking.py

```diff
@@ -8,10 +8,10 @@ class StockPicking(models.Model):
     _inherit = "stock.picking"
 
     def _check_backorder(self):
-        self.ensure_one()
-        # If strategy == 'manual', let the normal process going on
-        if self.picking_type_id.backorder_strategy == "manual":
-            return super(StockPicking, self)._check_backorder()
+        for picking in self:
```

Comment:
This is wrong. The guts of it is as I can tell is we are supposed to return a list of pickings if the backorder_strategy is manual and nothing if not. In theory multiple pickings could have multiple types and multiple strategies. Therefore what needs to happen is pretty much something like in pseudo code.

```
self = self.filtered(<backorder_strategy> == 'manual')
return super()
```